DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/ViewTreeSnapshotBuilder.swift

```diff
@@ -8,6 +8,7 @@
 import Foundation
 import UIKit
 import WebKit
+import DatadogInternal
```

Comment:
Is this import needed?

Reply:
It's because `SessionReplayImagePrivacyLevel` belongs to `DatadogInternal` at this point.

I'm revisiting this dependency. It might be wrong.